config/snowflake.spc

```diff
@@ -0,0 +1,46 @@
+connection "snowflake" {
+  plugin = "snowflake"
+
+  # Your Snowflake Account
+  # account = "xy12345"
+
+  # The user of your Snowflake Account
+  # user = "steampipe"
+
+  # The region id for Snowflake Account
+  # region = "ap-south-1.aws"
+
+  # Optional; The role that should be used by steampipe for the user.
+  # If not mentioned will use the default role for the User
+  # role = "ACCOUNTADMIN"
+
+  # Authentication to snowflake can be done in below ways
+  # 1. Using Password
+  # 2. Key Pair Authentication
+  # 3. OAuth Access Token
+  # 4. OAuth Refresh Token
+
+  # 1.1 Authentication using password
+  # The password for your Snowflake Account
+  # password = "~dummy@pass"
+
+  # 1.2 Key Pair Authentication
+  # https://docs.snowflake.com/en/user-guide/key-pair-auth.html
+  # private_key_path       = "/Users/lalitbhardwaj/Turbot/prod/my_sample_codes/Go_Basics/snowflake/rsa_key.p8"
+  # private_key_passphrase = "abcde"
+
+  # OR use private key directly
+
+  # private_key = "-----BEGIN ENCRYPTED PRIVATE KEY-----\nMIIFHzBJ....au/BUg==\n-----END ENCRYPTED PRIVATE KEY-----"
+  # private_key_passphrase = "abcde"
+
+  # 1.3 OAuth Access Token
+  # oauth_access_token = "...."
```

Comment:
What does an OAuth access token look like? We should use a similar format as the example